entity-framework/core/what-is-new/ef-core-8.0/breaking-changes.md

```diff
@@ -409,3 +410,30 @@ The old behavior was a bug and prevented properties that match the synthesized k
 #### Mitigations
 
 [Exclude the property from the model](xref:core/modeling/entity-properties#included-and-excluded-properties) if its value should not be persisted.
+Additionally, you can disable this behavior entirely by setting `Microsoft.EntityFrameworkCore.Issue31664` AppContext switch to `true`, see [AppContext for library consumers](/dotnet/api/system.appcontext#ForConsumers) for more details.
+
+```C#
+AppContext.SetSwitch("Microsoft.EntityFrameworkCore.Issue31664", isEnabled: true);
+```
+
+<a name="compiled-relational-model"></a>
+
+### Relational model is generated in the compiled model
+
+[Tracking Issue #24896](https://github.com/dotnet/efcore/issues/24896)
+
+#### Old behavior
+
+Previously, the relational model was computed at run-time even when using a compiled model.
+
+#### New behavior
+
+Starting with EF Core 8.0, the relational model is part of the generated compied model. However, for particularly large models the generated file may fail to compile.
+
+#### Why
+
+This was done to further improve startup time.
+
+#### Mitigations
+
+Edit the generated `*ModelBuilder.cs` file and remove the line `AddRuntimeAnnotation("Relational:RelationalModel", CreateRelationalModel());` as well as the method `CreateRelationalModel()`.
```

Comment:
@austin-patterson

Reply:
Note that this is not necessary when using EF9